app/client/src/skychat/SkyChatClient.js

```diff
@@ -46,34 +46,44 @@ export class SkyChatClient extends EventEmitter {
         this.on('roll', this.onRoll.bind(this));
         this.on('ytapi:search', this.onYtApiSearchResults.bind(this));
 
-        this.on('ping', this.sendPong.bind(this));
+        this.on('pong', this.onPong.bind(this));
+        this.on('ping', this.onPing.bind(this));
     }
 
     /**
      * Send a ping
      */
     sendPing() {
 
+        return;
+
         // If websocket is not open, ignore
```

Comment:
I disabled the ping mechanism since it actually had a negative impact on performances.
This needs to be investigated and fixed, I'll make a separated issue for this